assets/src/edit-story/app/story/actions/useDeletePage.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * WordPress dependencies
+ */
+import { useCallback } from '@wordpress/element';
+
+// @todo this might need more appropriate naming depending on how the deleting Page logic will be.
+// Delete the currently selected Page.
+function useDeletePage( {
```

Comment:
`useDeleteCurrentSelectedPage`?

Reply:
There's the state value for `currentPage`, however, a Page isn't actually added to a selection -- the comment above saying "currently selected Page" is a bit misleading. `useDeleteCurrentPage` would be the most consistent with the `useState` values, I suppose.

Will update this, thanks for noting that.

Reply:
Some jsdocs would be good here too.